gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -34,6 +34,7 @@
             "PreferSafeLoggingPreconditions",
             "StrictUnusedVariable",
             "StringBuilderConstantParameters",
+            "ThrowError",
```

Comment:
I'm a little worried promoting this straight away may lead to unexpected/unintended code rewriting - could we do a trial run on a few repos first?